### ISSUE TITLE
Increment/decrement should be outputted and independent from the one from assign and capture

### ISSUE THREAD

**Issue by adisuryadi** [harttle/liquidjs#76]
It seems that the increment/decrement affect the same variable name that created using assign or capture, it should instead independent. Their incremented values also not being outputted.

https://shopify.github.io/liquid/tags/variable/

```
{% assign var = 10 %}
{% increment var %}
{% increment var %}
{% increment var %}
{{ var }}
```
actual:
```
13
```
expected:
```
0 1 2 10
```

Thanks!

**Comment by harttle** [comment]
fixed on @5.0.2

**Comment by adisuryadi** [comment]
thanks!
